crontabber/tests/test_crontabber.py

```diff
@@ -330,6 +370,11 @@ def test_pop(self):
         popped = self.database.pop('foo', 'default')
         eq_(popped, 'default')
         assert_raises(KeyError, self.database.pop, 'bar')
+        try:
+            del self.database['bar']
+            assert False, "it should have raise a KeyError on del"
+        except KeyError:
+            pass
```

Comment:
Oh, I might take it back. Looks like nose pep8-ifies unittest testcase things. So you might also be able to do:

```
with assert_raises(KeyError):
    del self.database['bar']
```

Definitely worth trying.